Cesium3DTilesSelection/src/TilesetJsonLoader.cpp

```diff
@@ -488,35 +488,41 @@ std::optional<Tile> parseTileJsonRecursively(
     }
   }
 
-  // determine if tile points to implicit tiling extension
-  const auto extensionIt = tileJson.FindMember("extensions");
-  bool hasImplicitContent = false;
-  if (extensionIt != tileJson.MemberEnd()) {
+  // determine if tile is an implicit tile
+  const rapidjson::Value* implicitTilingJson = nullptr;
+  const auto implicitTilingIt = tileJson.FindMember("implicitTiling");
+  if (implicitTilingIt != tileJson.MemberEnd() &&
+      implicitTilingIt->value.IsObject()) {
     // this is an external tile pointing to an implicit tileset
-    const auto& extensions = extensionIt->value;
-    const auto implicitExtensionIt =
-        extensions.FindMember("3DTILES_implicit_tiling");
-    hasImplicitContent = implicitExtensionIt != extensions.MemberEnd() &&
-                         implicitExtensionIt->value.IsObject();
-    if (hasImplicitContent) {
-      // mark this tile as external
-      Tile tile{&currentLoader, TileExternalContent{}};
-      tile.setTileID("");
-      tile.setTransform(tileTransform);
-      tile.setBoundingVolume(tileBoundingVolume);
-      tile.setViewerRequestVolume(tileViewerRequestVolume);
-      tile.setGeometricError(tileGeometricError);
-      tile.setRefine(tileRefine);
-
-      parseImplicitTileset(
-          implicitExtensionIt->value,
-          contentUri,
-          tile,
-          currentLoader);
-
-      return tile;
+    implicitTilingJson = &implicitTilingIt->value;
+  }
+  if (!implicitTilingJson) {
```

Comment:
Changing this to an `else` works for me -- can you double-check if it works for you?